mateo-mustapic/assignments/liferay-app/sdk/portlets/todo-portlet/docroot/header.jspf

```diff
@@ -0,0 +1,34 @@
+<%@ include file="init.jsp" %>
```

Comment:
JSPFs are a direct include, this means you are doubling code in init.jsp. Don't include it.